### PR TITLE
Add @juanmaguitar as codeowners of gutenberg docs

### PR DESCRIPTION
**Pull request by juanmaguitar**
Adding @juanmaguitar as codeowners of gutenberg docs

### CODE REVIEW

File: .github/CODEOWNERS
Comment (ocean90) [inline .github/CODEOWNERS]: This causing an error in the file:

```
Unknown owner on line 2: make sure @mburridge exists and has write access to the repository
```

**Comment by juanmaguitar** [comment]
@ocean90 I've removed  @mburridge from this PR to unblock it. We will create another PR to add @mburridge to this list so we can deal with this issue apart

**Comment by mburridge** [comment]
@juanmaguitar is removing me the solution? Do I need to be granted permissions on this repo?

**Comment by mburridge** [comment]
It looks like I need to be added to the Gutenberg team. How do I request that?

**Comment by juanmaguitar** [comment]
> is removing me the solution? 

No, I don't think this is the final solution. But I think the best approach here is to create a separate PR to add your name to the CODEOWNERS so we can have a separated conversation to solve the issue that is causing this error

> Do I need to be granted permissions on this repo?

TBH, I don't know what's the cause of this error or the type of permissions we should have but I think It's best to deal with that in a separated PR. Can you create it to move this conversation there?

**Comment by mburridge** [comment]
I've requested membership of the Gutenberg team. I'll create a new PR once I've been granted membership.

**Comment by juanmaguitar** [comment]
> I've requested membership of the Gutenberg team. I'll create a new PR once I've been granted membership.

Awesome. Thanks @mburridge

**Review by cbravobernal — APPROVED** [review]

**Pull request merged**
